test/runBTagAnalyzer_cfg.py

```diff
@@ -397,7 +397,8 @@
     SelectEvents   = cms.untracked.PSet( SelectEvents = cms.vstring('p') ),
     # save PAT Layer 1 output; you need a '*' to
     # unpack the list of commands 'patEventContent'
-    outputCommands = cms.untracked.vstring('drop *', *patEventContent)
+#    outputCommands = cms.untracked.vstring('drop *', *patEventContent)
```

Comment:
This does not have to be modified since in the end no EDM output is produced (note `del process.out` at the bottom of the cfg file). The output module is only needed for PAT to be configured correctly.